src/main/java/com/box/sdk/BoxGroup.java

```diff
@@ -163,6 +163,34 @@ public BoxGroup(BoxAPIConnection api, String id) {
         };
     }
 
+    /**
+     * Gets an iterable of all the groups in the enterprise that are starting with the given name string.
+     * @param  api the API connection to be used when retrieving the groups.
+     * @param  name the name prefix of the groups. If the groups need to searched by full name that has spaces,
+     *              then the parameter string should have been wrapped with "".
+     * @param fields the fields to retrieve.
+     * @return     an iterable containing info about all the groups.
+     */
+    public static Iterable<BoxGroup.Info> getAllGroupsByName(final BoxAPIConnection api, String name,
```

Comment:
Can you please have the method above this one call this one?

```
public static Iterable<BoxGroup.Info> getAllGroupsByName(final BoxAPIConnection api, String name) {
     return getAllGroupsByName(api, name, null);
 }
```

Reply:
@wadahiro Can you make this change please?  Then I'll be able to approve.  Thanks!

Reply:
@wadahiro Please disregard - my system wasn't refreshing or something.  I see the change now.  Thanks!